api/src/test/resources/org/openmrs/api/include/UserServiceTest.xml

```diff
@@ -34,9 +34,13 @@
   <person person_id="5507" gender="M" dead="false" creator="1" date_created="2008-08-15 15:57:09.0" voided="false" uuid="dc99369f-f88e-4d54-be56-100e1ff45cf0"/>
   <person_name person_name_id="6" preferred="true" person_id="5507" given_name="Hank" middle_name=" " family_name="Williams" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="642e1d7f-00b8-45db-9705-b68341e61782"/>
   
+  <person person_id="5508" gender="M" dead="false" creator="1" date_created="2008-08-15 15:57:09.0" voided="false" uuid="dc99369fbvjhbj-njhgvj97-vc6-0nn7"/>
+  <person_name person_name_id="7" preferred="true" person_id="5508" given_name="Super" middle_name=" " family_name="User" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="6jhloijo-jhbjh877b-jghvky-m888hhy6"/>
+  
   <users user_id="5505" person_id="5505" system_id="6-7" username="incorrectlyhashedSha1" password="4a1750c8607dfa237de36c6305715c223415189" salt="c788c6ad82a157b712392ca695dfcf2eed193d7f" secret_question="" creator="1" date_created="2008-08-15 15:57:09.0" changed_by="1" date_changed="2008-08-18 11:51:56.0" retired="false" retire_reason="" uuid="013c49c6-e132-11de-babe-001e378eb67e"/>
   <users user_id="5506" person_id="5506" system_id="7-5" username="correctlyhashedSha1" password="4a1750c8607d0fa237de36c6305715c223415189" salt="c788c6ad82a157b712392ca695dfcf2eed193d7f" secret_question="" creator="1" date_created="2008-08-15 15:57:09.0" changed_by="1" date_changed="2008-08-18 11:51:56.0" retired="false" retire_reason="" uuid="06d05314-e132-11de-babe-001e378eb67e"/>
   <users user_id="5507" person_id="5507" system_id="8-3" username="userWithSha512Hash" password="74e3ed4f9e5b955de443feabc9f12ecd291ac6a7eaa2f09a72a5c811ec618b044d7131f687c0a0465fc0c660d09fd245e410e12ffe77d78b2a0c504b40afa202" salt="1c30292f0b438b49b181950f3e831c6a9f38ef1c7d559fca566e8cd6b4efb417150585d4c3345f0fb0b520fc0884ad82ea7c1a28f48c2c562106a3d110716cfc" secret_question="secret" secret_answer="answer" creator="1" date_created="2008-08-15 15:57:09.0" changed_by="1" date_changed="2008-08-18 11:51:56.0" retired="false" retire_reason="" uuid="0bc18050-e132-11de-babe-001e378eb67e"/>
+  <users user_id="5508" person_id="5508" system_id="9-9" username="userThatReplacesDaemonForAmbiguousUseCases" password="eeeda5c0cc3837151b2d61cfeab54a91kjbkjb88" salt="42af4c437a47cd778a54flh6gy" secret_question="" secret_answer="" creator="1" date_created="2008-08-15 15:57:09.0" changed_by="1" date_changed="2008-08-18 11:51:56.0" retired="false" retire_reason="" uuid="0bc18050-e132-11de-11kb-hjhh87hui"/>
```

Comment:
Well, I added this User because of the effect of filtering of the Daemon User it fails the the test that @dkayiwa  complained about ( 	publicresolveUserId_shouldReturnNullForAmbiguousUsersUsingFirstAndLastNameGivenUserIDIsNull() ).